Tests/BinaryCompatibility.lean

```diff
@@ -31,9 +31,7 @@ def fname (s : String) : String :=
 partial def testGeneric : String → ByteArray → ByteArray → TestSeq
   | mod, rs, os =>
     let str := s!"Binary representation is compatible with {fname mod}
-    === CODE ===
-    {mod}
-    ||| END OF CODE |||"
+    {mod}"
```

Comment:
Not sure this is a good change.
Makes it harder to see where code blocks start and end.

Reply:
I find it's on the contrary. Compare on readability:

![2023-03-06-213338_571x333_escrotum](https://user-images.githubusercontent.com/16266191/223224455-3687e048-5873-4ffd-ac6d-9eaf749bb6bf.png)
![2023-03-06-213328_573x432_escrotum](https://user-images.githubusercontent.com/16266191/223224457-899da7b0-6f30-4d20-856b-069ce6b172b0.png)

Personally at least, I get lost in the surrounding text.